region/norfair/west.json

```diff
@@ -1542,6 +1546,7 @@
                     {"or": [
                       "Grapple",
                       "SpaceJump",
+                      "canSpringBallJumpMidAir",
```

Comment:
I didn't exclude it by mistake, it's missing about half a tile of height off a crouch jump. What did you do?

Reply:
Though it is doable with hjb and that doesn't seem to be there.

Reply:
In my notes the HiJump is included just forgot to put it in the code.